src/App.js

```diff
@@ -0,0 +1,10 @@
+import { CalculatorDesign } from "./components/Calculator";
+function App() {
+  return (
+    <div className="App">
+      <CalculatorDesign/>
+    </div>
+  );
+}
+
+export default App;
```

Comment:
kindly change the App into a class component, as mentioned in the project requirements all components should be class based 👍 😄 
PS: You will refactor them into functional components in upcoming projects 👍